wireguard_client/config.json

```diff
@@ -40,15 +40,12 @@
          "post_up": "iptables -t nat -A POSTROUTING -o wg0 -j MASQUERADE",
          "post_down": "iptables -t nat -D POSTROUTING -o wg0 -j MASQUERADE"
       },
-      "peer":{
+      "peers":[{
          "public_key": "",
          "pre_shared_key": "",
          "endpoint": "xxxxxx.duckdns.com",
-         "allowed_ips":[
-            "10.6.0.0/24"
-         ],
-         "persistent_keep_alive": "25"
```

Comment:
this param is very important to be keep

Reply:
It works just fine without `persistent_keep_alive`, if you only plan to make outbound connections it works. But you are correct that it's probably a good idea to make sure it's set.